<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCreateNote</class>
 <widget class="QDialog" name="DialogCreateNote">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="TitleWidget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>781</width>
     <height>41</height>
    </rect>
   </property>
   <widget class="QLabel" name="HeadlineLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="DialogCreatorWidget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>781</width>
     <height>461</height>
    </rect>
   </property>
   <widget class="QLabel" name="WhatIsYourNewNoteNameLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NoteNamelineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>761</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="AddNoteButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>761</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="NoteBodytextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>761</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="WhatIsYourNewNoteDescription">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="SelectParentNotebook">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ParentNotebookSelector">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>761</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>781</width>
     <height>91</height>
    </rect>
   </property>
   <widget class="QLabel" name="DialogDescriptionText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
